cypress.config.ts

```diff
@@ -4,8 +4,14 @@ import { defineConfig } from 'cypress'
  * Example: https://github.com/cypress-io/cypress-realworld-app/blob/develop/cypress.config.js
  */
 export default defineConfig({
+  env: {
+    // LOGIN_SESSION_COOKIE: "8741b1d9-bc1d-4bba-9e0b-6cb8d551978c",
+    LOGIN_SECURE_COOKIE: "8741b1d9-bc1d-4bba-9e0b-6cb8d551978c",
```

Comment:
it would require extra module to import local env into Cypress. since the values are only used in cypress, prob it's okay to put a placeholder directly in this config file. the corresponding values are added during docker startup process.